tests/instances.spec.ts

```diff
@@ -3,10 +3,117 @@ import {
   createInstance,
   deleteInstance,
   randomInstanceName,
+  renameInstance,
 } from "./instance-helpers";
+import {
+  assertCode,
+  assertReadMode,
+  setCodeInput,
+  setInput,
+  setOption,
+} from "./configuration-helpers";
+import { TIMEOUT } from "./constants";
 
 test("instance create and remove", async ({ page }) => {
   const instance = randomInstanceName();
   await createInstance(page, instance);
   await deleteInstance(page, instance);
 });
+
+test("instance rename", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+
+  const newName = instance + "-rename";
+  await renameInstance(page, instance, newName);
+
+  await deleteInstance(page, newName);
+});
+
+test("instance configuration edit", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+
+  await page.getByPlaceholder("Search").click();
+  await page.getByPlaceholder("Search").fill(instance);
+  await page.getByRole("link", { name: instance }).first().click();
+  await page.getByTestId("tab-link-Configuration").click();
+  await page.getByRole("button", { name: "Edit instance" }).click();
+
+  await page.getByText("Resource limits").click();
+  await setOption(page, "Memory swap", "true");
+  await setOption(page, "Disk priority", "1");
+  await setInput(page, "Max number of processes", "Enter number", "2");
+
+  await page.getByText("Security policies").click();
+  await setOption(page, "Prevent the instance from being deleted", "false");
+  await setOption(page, "Run the instance in privileged mode", "true");
+  await setOption(
+    page,
+    "Prevent instance file system from being UID/GID shifted",
+    "true"
+  );
+  await setInput(page, "Base host id", "Enter ID", "11");
+  await setInput(page, "Idmap size", "Enter number", "22");
+  await setOption(page, "Unique idmap usage", "true");
+  await setOption(page, "Allow /dev/lxd in the instance", "true");
+  await setOption(page, "Make /1.0/images API available", "true");
```

Comment:
Maybe we should add another test where a VM is created, so that it allows to test also the missing config here (which is for VMs only)?

Reply:
VMs are much slower to create, so I avoided adding tests for them.

Reply:
A VM for alpine should be reasonably fast to create, no? At least so it seems on my (extremely slow) local environment...